Google.Api.Generator.IntegrationTests/ShowcaseTestBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.Grpc;
+using Google.Api.Gax.Grpc.Rest;
+using Grpc.Core;
+using System;
+using Xunit;
+
+namespace Google.Api.Generator.IntegrationTests
+{
+    /// <summary>
+    /// Base class for all Showcase tests.
+    /// </summary>
+    public abstract class ShowcaseTestBase<TClient, TBuilder>
```

Comment:
Just curious if you considered this being a base fixture class?
Then there's a rest fixture and a gRPC fixture and more specific fixtures if we needed.

It's fine as is though.

Reply:
We could do that, but I suspect we'd end up with more classes that way, and pointless constructors accepting fixtures just to achieve what we've already achieved here. If we end up needing to do more than create a client though, it would make sense.